cmd/burrow/commands/deploy.go

```diff
@@ -49,7 +49,7 @@ func Deploy(output Output) func(cmd *cli.Cmd) {
 			"default number of concurrent playbooks to run if multiple are specified")
 
 		addressOpt := cmd.StringOpt("a address", "",
-			"default address to use; operates the same way as the [account] job, only before the deploy file is ran")
+			"default address (or validator name) to use; operates the same way as the [account] job, only before the deploy file is ran")
```

Comment:
This does not have to be a validator name; it can be any account name. The deploy script could just call a contract function to get the return value, for example.

Reply:
Ahh okay, could you comment on [this](https://github.com/hyperledger/burrow/issues/983) issue as well?